src/python/build_management/build_manager.py

```diff
@@ -692,7 +722,11 @@ def build_dir(self):
   def setup(self):
     """Sets up build with a particular revision."""
     self._pre_setup()
-    environment.set_value('BUILD_URL', self.build_url)
+    if self.build_prefix:
+      environment.set_value('%s_BUILD_URL' % self.build_prefix, self.build_url)
+    else:
+      environment.set_value('BUILD_URL', self.build_url)
```

Comment:
nit: consider having self.env_prefix and use here and everywhere else that does similar things.